testRelease fails in non configured environment

I would be good to have testRelease in shipkit-example up and running to new contributors to test their changes before making a PR. Currently, on a "fresh environment" it fails:

```
gw testRelease
(...)
[releaseNeeded] :gitPush
[releaseNeeded]   'git push' does not use GitHub write token because it was not specified
[releaseNeeded]   Executing:
[releaseNeeded]     git push https://github.com/mockito/shipkit-example.git v0.16.8 master --dry-run
[releaseNeeded] :gitPush FAILED
[releaseNeeded] 
[releaseNeeded] FAILURE: Build failed with an exception.
[releaseNeeded] 
[releaseNeeded] * What went wrong:
[releaseNeeded] Execution failed for task ':gitPush'.
[releaseNeeded] > Execution of command failed (exit code 128):
[releaseNeeded]     git push https://github.com/mockito/shipkit-example.git v0.16.8 master --dry-run
[releaseNeeded]     Captured command output:
[releaseNeeded]   fatal: could not read Username for 'https://github.com': No such device or address
```

Even `-x gitPush` doesn't help as gitPush in a dry-run mode is executed in a Gradle subprocess.

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -22,9 +22,11 @@ This will install the artifacts in local maven repository for easy sharing.
 Notice the version you're building in the build output.
 2. Clone shipkit-example repo and ensure that 'shipkit-example/build.gradle' file uses the correct version of shipkit (declared at the top of build.gradle).
 It should use the same version that was built in the previous step.
-3. Basic testing (for most contributors):
- - Smoke test (no tasks are run): ```./gradlew testRelease -m```
- - Test most things, without actually making any code pushes/publications: ```./gradlew testRelease -x gitPush -x bintrayUpload```
+3. Basic testing for contributors:
+ - Smoke test (no tasks are run): ```./gradlew performRelease -m```
+ - Test most things, without actually making any code pushes/publications:
+ ```./gradlew releaseNeeded performRelease releaseCleanUp -x gitPush -x bintrayUpload -PdryRun```
+ TODO: add "contributorTestRelease" task that wraps above similar to "testRelease" task.
  - Release notes content: ```./gradlew updateReleaseNotes -Ppreview```
     To generate sizable release notes content, before running 'updateReleaseNotes' you can downgrade the 'previousVersion' in 'version.properties'.
     Release notes are generated from 'previousVersion' to current 'version' as declared in 'version.properties' file.

--- a/build.gradle
+++ b/build.gradle
@@ -5,7 +5,7 @@ buildscript {
         maven { url "https://plugins.gradle.org/m2/" }
     }
     dependencies {
-        classpath "org.shipkit:shipkit:0.9.65"
+        classpath "org.shipkit:shipkit:0.9.81"
         classpath 'ru.vyarus:gradle-animalsniffer-plugin:1.3.0'
     }
 }

--- a/subprojects/testDownstream/testDownstream.gradle
+++ b/subprojects/testDownstream/testDownstream.gradle
@@ -3,5 +3,4 @@ apply plugin: 'base' // for clean task, TODO ms: push it to the plugin, make 'ch
 apply plugin: 'org.shipkit.test-downstream'
 
 testDownstream.addRepository("https://github.com/mockito/shipkit-example")
-testDownstream.addRepository("https://github.com/mockito/shipkit-bootstrap")
 testDownstream.addRepository("https://github.com/mockito/mockito")

